dotnet/src/Connectors/Connectors.MistralAI.UnitTests/MistralTestBase.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Connectors.MistralAI.Client;
+using Microsoft.SemanticKernel.Http;
+using Xunit;
+
+namespace SemanticKernel.Connectors.MistralAI.UnitTests;
+public abstract class MistralTestBase : IDisposable
```

Comment:
Have you considered adding this to TestUtilities ? 

We have in the `/InternalUtilities/samples/InternalUtilities/BaseTest.cs` may be interesting to bring another copy to Unit Tests.

```
  <ItemGroup>
    <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/test/AssertExtensions.cs" Link="%(RecursiveDir)%(Filename)%(Extension)" />
    <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/test/BaseTest.cs" Link="%(RecursiveDir)%(Filename)%(Extension)" />
  </ItemGroup>
```

Reply:
I'll look into this in a sperate PR